src/components/__test__/CountryList.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render } from '@testing-library/react';
+import store from '../../Redux/configureStore';
+import CountryList from '../CountryList';
+
+describe('It renders', () => {
+  it('CountryList correctly', () => {
+    const tree = render(
+      <Provider store={store}>
+        <CountryList />
+      </Provider>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] One of the requirements for this project is to use **_React Testing Library_** to write integration tests for your application. **_React Testing Library_** promotes testing your application as a user would interact with it, rather than focusing on implementation details. Kindly implement this requirement. You can read this [article](https://www.freecodecamp.org/news/react-testing-library-tutorial-javascript-example-code/) to learn how to use **_React Testing Library_**. 👍 😎